tfconfig/test-fixtures/basics/basics.out.json

```diff
@@ -1,10 +1,11 @@
 {
   "path": "test-fixtures/basics",
-
   "required_providers": {
-    "null": []
+    "null": {
+      "source": "",
```

Comment:
Could/should we `omitempty` this thing so that it's absent when not set? Not a big deal, but would make the result a little more compact and I think it's consistent with how other parts of this package treat unset things. :thinking:

Reply:
Abso-I-am-embarassed-lutely!